drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties

```diff
@@ -20,7 +20,11 @@ EditMenu.paste=Paste
 EditMenu.deleteCells=Delete cell(s)
 EditMenu.deleteColumns=Delete column(s)
 EditMenu.deleteRows=Delete row(s)
-EditMenu.otherwise=Otherwise
+EditMenu.otherwise=Insert "any other" value
+EditMenu.otherwiseDescription=<p>For condition columns in Guided Decision Tables, you can apply an "any other" value to table cells within the column if the following parameters are set: \
```

Comment:
I have mixed feelings about putting HTML tags here. But, I think that it's the best approach in this context.

An alternative would be the use of something like this:
```
EditMenu.otherwiseDescription1=...
EditMenu.otherwiseDescription2=...
EditMenu.otherwiseDescription3=...
```
I think that this alternative pollutes the code of the View.

So, good approach :-)